packages/studio-ui/src/web/components/Layout/TopNav/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { Button, Position } from '@blueprintjs/core'
+import { lang, ToolTip, utils } from 'botpress/shared'
+import cx from 'classnames'
+import React, { FC } from 'react'
+import { RiLayoutLeftLine, RiLayoutRightLine, RiLayoutBottomLine } from 'react-icons/ri'
+import { connect } from 'react-redux'
+import { toggleBottomPanel, toggleExplorer } from '~/actions'
+
+import { RootReducer } from '../../../reducers'
+import EnterPriseTrial from './EnterpriseTrial'
+import RightToolBar from './RightToolbar'
+
+import style from './style.scss'
+
+type Props = ReturnType<typeof mapStateToProps> & typeof mapDispatchToProps
+
+const Toolbar: FC<Props> = props => {
+  // We cannot use setEmulatorOpen as it's actually the state is actually controlled by the message passed from the iframe
+  // Will need some refactor for the emulator
+  // This should be moved this somewhere else, it seems to be all around the app
+  const toggleEmulator = () => {
+    window.botpressWebChat.sendEvent({ type: 'toggle' })
+  }
+
+  return (
+    <nav className={style.topNav}>
+      <EnterPriseTrial />
+      <div className={style.layoutControls}>
+        <ToolTip
+          content={lang.tr('topNav.toggleExplorer', { shortcut: `${utils.shortControlKey} B` })}
+          position={Position.BOTTOM}
+        >
+          <Button
+            onClick={props.toggleExplorer}
```

Comment:
For these buttons you can add the `minimal` prop ([reference](https://blueprintjs.com/docs/#core/components/button)). Then you need a lot less css

```scss
.layoutControls {
    button {
      color: var(--c-text--disabled);

      &:hover,
      &.active {
        color: var(--shark);
      }

      &:hover {
        background: var(--bg);
      }
    }
  }
```